cli/credentials.ts

```diff
@@ -5,6 +5,27 @@ import { actuallyResolve } from "df/cli/util";
 import { dataform } from "df/protos/ts";
 
 export function getBigQueryCredentials(): dataform.IBigQuery {
+  const locationIndex = selectionQuestion("Enter the location of your datasets:", [
+    "US (default)",
+    "EU",
+    "other"
+  ]);
+  let location = locationIndex === 0 ? "US" : "EU";
+  if (locationIndex === 2) {
+    location = question("Enter the location's region name (e.g. 'asia-south1'):");
+  }
+  const isApplicationDefaultOrJSONKeyIndex = selectionQuestion("Do you wish to use Application Default Credentials or JSON Key:", [
+    "ADC (default)",
+    "JSON Key"
+  ]);
+  if (isApplicationDefaultOrJSONKeyIndex === 0)  {
+    const projectId = question("Enter your billing project:");
+    return {
+      projectId: projectId,
+      credentials: "",
```

Comment:
can you leave a comment on the relevant protobuf field indicating that if it's unset, this implies that ADC should be used instead? https://github.com/dataform-co/dataform/blob/master/protos/profiles.proto#L46